### DISCUSSION TITLE
Option to set screen positions on widescreen output

### DISCUSSION THREAD

**Discussion by Baracudus** [diasurgical/DevilutionX#6242] [Ideas]
When playing on a widescreen monitor, and the character screen or inventory (or any other screen like theese) is openen, they appear on the left an right side of the monitor. My Idea is to give an option to center these screens in the middle of the monitor for better accessability. May this be possible?

screens left/right (as today):
![screens aside](https://github.com/diasurgical/devilutionX/assets/71731621/960a4bce-de64-4b90-b343-463b307b9847)

screens centered
![screens centered](https://github.com/diasurgical/devilutionX/assets/71731621/bb52c2ae-f1eb-4440-af80-a5432a042f67)

**Comment by StephenCWills** [comment]
We do that automatically when using the touchscreen controls because the left and right panels will be less likely to overlap with the on-screen controls if we center them like this. We don't have an option for it, though.

https://github.com/diasurgical/devilutionX/blob/81a034132a16644a45ceb1d8e1ef0800cb794c71/Source/control.cpp#L665-L679

**Reply by Baracudus** [reply]
Ok, nice that It's already implemented. Maybe there will be an option in the future. 😁

**Reply by julealgon** [reply]
@Baracudus the idea in the long-term is to have the UI be customizable and adjustable but the team is small so its tough to prioritize everything. Might take a while until the project reaches that state but it was definitely thought about 😉
